File: HOPE_13/Models/Hotel.cs
using System.Collections.Generic;

namespace HOPE_13.Models
{
    public class Hotel
    {
        public int Id { get; set; }
        public string hotel { get; set; }
        public int Cost { get; set; }
        public int QuantityNight { get; set; }
        public string TypeOfRooms { get; set; }
        public string Nutrition { get; set; }
        public string Information { get; set; }

        public int? ToursId { get; set; }
        public Tour Tours { get; set; }
        public ICollection<ImageHotel> ImageHotels { get; set; }

    }
}

File: HOPE-SPA/src/app/_models/hotel.ts
import { HotelImages } from './hotelImages';

export interface Hotel {
    id: number;
    hotel: string;
    Cost: number;
    QuantityNight: number;
    TypeOfRooms: string;
    Nutrition: string;
    Information: string;
    ImageUrl1: string;
    hotelImages: HotelImages[];
    // ToursId: number;
}
